uvicorn/config.py

```diff
@@ -355,28 +341,38 @@ def setup_event_loop(self):
             loop_setup()
 
     def bind_socket(self):
-        family, sockettype, proto, canonname, sockaddr = socket.getaddrinfo(
-            self.host, self.port, type=socket.SOCK_STREAM
-        )[0]
-        sock = socket.socket(family=family, type=sockettype)
+        host = self.host
```

Comment:
no big deal but why we dont keep self.host / self.port along the way in the method, that doesnt seem necessary or I am missing something ?